packages/contracts/src/SubdomainController.sol

```diff
@@ -2,45 +2,34 @@
 pragma solidity ^0.8.17;
 
 import {INameWrapper} from "@ens-contracts/wrapper/INameWrapper.sol";
+import {ENSRegistry} from "@ens-contracts/registry/ENSRegistry.sol";
 import {Resolver} from "@ens-contracts/resolvers/Resolver.sol";
 import {BytesUtils} from "@ens-contracts/utils/BytesUtils.sol";
 
 import {ENSHelper} from "../script/ENSHelper.sol";
-import {OffchainRegister} from "./interfaces/OffchainResolver.sol";
+import {
+    OffchainRegister,
+    RegisterRequest
+} from "./interfaces/OffchainResolver.sol";
 
 contract SubdomainController is OffchainRegister, ENSHelper {
 
     using BytesUtils for bytes;
 
     uint256 public price;
```

Comment:
Nenhuma func tirando o construtor altera "price". Permitir que alguem modifique isso é um possivel use case? Se for, vale ter uma func permisionada pra alterar isso.